assets/section-footer.css

```diff
@@ -1,11 +1,8 @@
-.footer {
+.footer-wrapper--border-top,
+.footer-wrapper--border-top .footer__content-bottom:not(:only-child) {
   border-top: 0.1rem solid rgba(var(--color-foreground), 0.08);
 }
 
-.footer:not(.color-background-1) {
-  border-top: none;
-}
```

Comment:
This means that it would have a visual impact on merchants' stores potentially where if they were using a `background-1` `color scheme` it would hide the separator lines. Or the other way around if we set them to `true` by default.
So any merchant using a `color scheme` other than `background-1` will now get some separator lines showing. 

Not a big issue, just wanting to flag this as something that we will need to be aware when doing new releases.